webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -114,12 +123,37 @@ class ManageManifestModal extends Component {
   manifestExists() {
     const { organization } = this.props;
 
-    return organization.owner_details && organization.owner_details.upstreamConsumer;
+    return (
+      organization.owner_details && organization.owner_details.upstreamConsumer
+    );
+  }
+
+  renderTaskStartedToast() {
+    const { taskDetails } = this.props;
+    const message = (
+      <span>
+        <span>
+          {__(`Task ${taskDetails.humanized.action} has started`)}
```

Comment:
Missing punctuation here.  Maybe make this "Task ____ has started.  Click here to go to the tasks page for the task."

Reply:
Watch out, you need to use `sprintf` here.